cwms-data-api/src/main/java/cwms/cda/api/watersupply/WaterUserUpdateController.java

```diff
@@ -96,11 +97,11 @@ public void handle(@NotNull Context ctx) {
             String formatHeader = ctx.header(Header.ACCEPT) != null ? ctx.header(Header.ACCEPT) : Formats.JSONV1;
             ContentType contentType = Formats.parseHeader(formatHeader, WaterUser.class);
             ctx.contentType(contentType.toString());
-            String newName = ctx.queryParam(NAME);
+            String newName = Objects.requireNonNull(ctx.queryParam(NAME));
```

Comment:
This will throw a NullPointerException if not null, which is handled as an internal error (error code 500) rather than a client error (error code 400). Instead, use the `requiredParam(ctx, NAME)`

Reply:
Swapped for requiredParam